src/Components/Blog.tsx

```diff
@@ -1,27 +1,142 @@
+                </p>
+                <img className="image-heading" src={llamas} alt="llamas" />
+                <p className="article-title text-center w-100"><strong>May 6: Antibodies of llamas proven to neutralize COVID-19</strong></p>
+                <p className="articles text-center w-100">
+                  Didn&apos;t think you see llamas on our newsletter, right? Well, scientists have long turned to llamas for antibody research. For example, studies were conducted for SARS, MERS, H.I.V, and influenza. These same antibodies were recently found to be able to neutralize the new virus, with the encouraging results published earlier this Tuesday.
+                  <a href="https://www.nytimes.com/2020/05/06/science/llama-coronavirus-antibodies.html">(NYTimes)</a>
+                </p>
+                <p className="article-title text-center w-100"><strong>May 5: U.K lockdown could be eased from Monday</strong></p>
+                <p className="articles text-center w-100">
+                  Prime Minister Boris Johnson said that the next phase of the governments strategy, easing lockdown, may begin on Monday. This is because U.K. is past its peak and hopes to rejuvenate the economy as soon as possible. Read more about their plan here!
+                  <a href="https://www.bloomberg.com/news/articles/2020-05-06/johnson-says-some-changes-to-u-k-lockdown-likely-from-monday"> (Bloomberg)</a>
+                </p>
+              </div>
+            </Section>
+            <Section title="Some Inspiration" table center>
+              <div>
+                <h1 className="section-heading text-center w-100">Some Inspiration!</h1>
+                <p className="message-heading text-center w-100">
+                  Featuring some amazing people and projects around the world
+                  <span role="img" aria-label="world"> 🌏</span>
+                </p>
+                <img className="image-heading" src={helpthehungry} alt="help the hungry" />
+                <p className="article-title text-center w-100"><strong>Help The Hungry: Top chefs provide free meals</strong></p>
+                <p className="articles text-center w-100">
+                  Chef in Schools, a charity co-founded by Nicole Pisani and Henry Dimbleby recruited staff from top restaurants to cook for 450 families. I can smell the food from here!
+                  <span role="img" aria-label="foods"> 🍛🍔🍣</span>
+                </p>
+                <p className="article-title text-center w-100"><strong>Virtual pub stand-up comedy</strong></p>
+                <p className="articles text-center w-100">
+                  The Covid Arms is a virtual comedy pub raising money for The Trussell Trust which provides food banks for the vulnerable. Imagine paying just £2 for laughter and the ability to contribute back to society!
+                  <span role="img" aria-label="sunglasses"> 😎</span>
+                </p>
+              </div>
```

Comment:
it really looks like you can separate this out into smaller components instead of duplicating code.

Reply:
sure, I could definitely try to do that, I just didnt change much of the example newsletter code from Rebeccas repo but ill try to break it up where I can